src/stacker/ConcreteStacker.ts

```diff
@@ -25,6 +25,27 @@ export class ConcreteStacker implements Stacker {
     this.repositoryUpdateListener(repo);
   };
 
+  private localRefPrefix : string = "refs/heads/";
+
+  private refIsLocal(refString: string) : boolean {
+    return refString.startsWith(this.localRefPrefix);
+  }
+
+  private localRefToBranchName(refString: string) : string {
+    if (!this.refIsLocal(refString)) return refString;
+    return refString.substr("refs/heads/".length);
+  }
+
+  private isSttackBranch(branch: BranchName) : boolean { 
+    if(branch.substr(0,6) === "sttack") return true; 
+    return false; 
+  }
+
+  private createSttackBranch(commit: Commit) : string {  // TODO: Do we need any parameters? Also what happens after we have a branch name? 
+    let branch : BranchName = 'sttack-${randomWords({ exactly: 4, join: "-" })}'; 
+    return branch
```

Comment:
I think you'll need to run Prettier on your code, but anyway you'll also need to use backticks instead of single quotes to interpolate that string.

```suggestion
  private createSttackBranch(commit: Commit): BranchName {
    return `sttack-${randomWords({ exactly: 4, join: "-" })}`;
```

> Also what happens after we have a branch name?

We can just ignore the commit and return the commit's stack attack branch